crates/libs/services/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "windows-services"
-version = "0.0.0"
+version = "0.24.0"
```

Comment:
Out of curiosity, why `0.24.0` for a new crate?

Reply:
I originally attempted to generate namespace-specific crates from metadata. That experiment shaped where we are today in terms of having `windows-bindgen` and other discrete crates. I will likely move all crates to 1.0 at some point and then these versions should be less confusing.

Reply:
To expand, the experiment generated and published crates like "windows-services" for the APIs in the "Windows.Services" namespace, and many others. The main problem with that approach is that Windows APIs lack robust layering and routinely have bidirectional dependencies which is incompatible with Cargo/Rust.

Reply:
I feel like we're talking past each other (or I'm missing a crucial detail to follow along). To rephrase the question: Why isn't `windows-services` introduced with version `0.1.0` (just like the other new `windows-threading` crate)? Why `0.24.0` specifically?

Reply:
Because that's impossible for the reason I mentioned. Have a look at the current version: https://crates.io/crates/windows-services

Reply:
Oh, now *that* makes perfect sense! I wasn't aware of that period of the evolution, that burnt through makeshift `cargo publish` crates that never were.

Thanks for following up, Kenny.